src/objects/objects.cc

```diff
@@ -5538,14 +5538,8 @@ Handle<Object> JSPromise::Reject(Handle<JSPromise> promise,
   promise->set_status(Promise::kRejected);
 
   {
-    // [TT-187] Disallow events in ReportPromiseReject if we did not report
-    // when recording.
-    bool allowed = !recordreplay::AreEventsDisallowed();
-    bool recordReport = allowed && recordreplay::RecordReplayValue(
-        "JSPromise::Reject has_handler", (uintptr_t)!promise->has_handler());
-    bool replayReport = !promise->has_handler();
-    v8::replayio::AutoMaybeDisallowEvents disallow(
-        allowed && !recordReport && replayReport, "JSPromise::Reject");
+    v8::replayio::AutoMaybeDisallowEvents disallow("JSPromise::Reject %d",
```

Comment:
comment to tag the linear issue?

Reply:
also it looks like the argument order is wrong (reversed) here.

Reply:
In some places, I don't want to add a tag, because its too generic.
But it often gets very tough to determine what is too generic and what isn't.
Maybe, we just have to always label them, and then (in the future) add smarter auto-bucketing, so we don't see the "wrong" bucket for crashes that happen to hit a "generic" (or generally speaking, unrelated) assert label... :thinking: